.changeset/tidy-zebras-poke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@scalar/api-reference': minor
```

Comment:
wanna make this a patch @cactysman ✨

Reply:
we can make it more clear in our contributing guide ✨

Reply:
can just hit commit on these requested changes :)